server/src/main/java/org/opensearch/identity/scopes/ScopeEnums.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.identity.scopes;
+
+public class ScopeEnums {
+
+    public enum ScopeArea {
+        APPLICATION,
+        CLUSTER,
+        EXTENSION_POINT,
```

Comment:
Would this look like `ExtensionPoint.ExtensionPoint.ActionPlugin` ?

Reply:
For now, that is what I have. I don't love it as a convention however. It feels like the Area is meaningless for the Extension Point Scopes but I don't know of an appropriate method of removing it while maintaining the Scope Schema.

Reply:
What about `ExtensionPoint.ActionPlugin.Allow`?  IMO we should be careful about these names, we will have to live with them for a long time!